lms/envs/aws.py

```diff
@@ -888,3 +888,7 @@
 ############## Settings for LMS Context Sensitive Help ##############
 
 DOC_LINK_BASE_URL = ENV_TOKENS.get('DOC_LINK_BASE_URL', DOC_LINK_BASE_URL)
+
+############## Settings for the Enterprise App ######################
+
+ENTERPRISE_ENROLLMENT_API_URL = ENV_TOKENS.get('ENTERPRISE_ENROLLMENT_API_URL', ENTERPRISE_ENROLLMENT_API_URL)
```

Comment:
Note that https://github.com/edx/configuration/blob/08486dd2623c3b861fda390ecc55a59e1241fe61/playbooks/roles/ecommerce/defaults/main.yml#L145 uses a different convention for the enrollment API URL. I don't think there's any conflict nor need to change this but it bugs me a little that multiple IDAs have to separately define "where to access the enrollment API" and that they do so differently.